README.md

```diff
@@ -60,15 +67,18 @@ from the latest checkpoint when a failure happens. The actions of flash checkpoi
 2. Persist the checkpoint to the storage once the training process fails.
 3. Load the checkpoint from the host memory after the training process restarts.
 
-After applying the fault tolerance and flash checkpoint of DLRover, **the overall goodput
-for the largest-scale training job using thousands of GPUs increased from 69% to 95%** .
-The goodput is the time spent computing useful new steps over the elapsed time of the training job.
-The downtime details are shown:
-
 <div align="center">
-<img src="docs/figures/dlrover-goodput-performance.jpg" alt="Editor" width="600">
+<img src="docs/figures/ft_llm_training/checkpoint_save_time.png" alt="Editor" width="396">
+<img src="docs/figures/ft_llm_training/checkpoint_load_time.jpg" alt="Editor" width="400">
+
+<text> The Performance of DLRover Flash Checkpoint to Save/Load GPT2-1.5B.</text>
 </div>
 
+The figure illustrates that the I/O time overhead to read checkpoint files
+when resuming by restarting training processes. With DLRover Flash Checkpoint,
+recovery directly from shared memory takes essentially
+on the order of seconds wich is much faster than SSD and NAS.
```

Comment:
recovery could be completed in the order of seconds by loading checkpoints directly from shared memory, which is much faster compared to loading checkpoints from SSD and NAS.